core/node/main/mainnet.cfg

```diff
@@ -1,5 +1,5 @@
-
 drand-server=api.drand.sh
 drand-pubkey=868f005eb8e6e4ca0a47c8a77ceaa5309a47978a7c71bc5cce96366b5d7a569937c529eeda66c7293784a9402801af31
 drand-genesis-time=1595431050
 drand-period=30
+
```

Comment:
Is it extra empty line? Please, check if there is 1 empty line and not 2 empty lines.